dashboard/client/src/pages/actor/ActorList.tsx

```diff
@@ -0,0 +1,59 @@
+import { Grid, Switch } from "@material-ui/core";
+import dayjs from "dayjs";
+import React, { useEffect, useState } from "react";
+import { API_REFRESH_INTERVAL_MS } from "../../common/constants";
+import ActorTable from "../../components/ActorTable";
+import { getActors } from "../../service/actor";
+import { Actor } from "../../type/actor";
+
+/**
+ * Represent the embedable actors page.
+ */
+const ActorList = ({ jobId = null }: { jobId?: string | null }) => {
+  const [autoRefresh, setAutoRefresh] = useState(true);
```

Comment:
let's get rid of this "auto refresh" button. We don't htink it's necessary. (Pages should just always auto-refresh)

Reply:
I may want to do this after the collapse & nav bar since this will ping many endpoints at once (and could be potentially heavy since they are not paginated yet). Would this be okay?

Reply:
removed auto refresh for both actor / pg